src/shared/database/schema.ts

```diff
@@ -20,43 +20,71 @@ export type JsonValue = JsonArray | JsonObject | JsonPrimitive;
 
 export type Timestamp = ColumnType<Date, Date | string, Date | string>;
 
-export interface DexExCandlesticks {
-  close: number;
-  direct_volume: number;
-  high: number;
-  id: Generated<number>;
-  low: number;
-  open: number;
-  swap_volume: number;
-}
-
 export const durationWindows = ['1m', '15m', '1h', '4h', '1d', '1w', '1mo'] as const;
 export type DurationWindow = (typeof durationWindows)[number];
 export const isDurationWindow = (str: string): str is DurationWindow =>
   durationWindows.includes(str as DurationWindow);
```

Comment:
question: are these constants and functions also generated by kysely? If not, can you move them somewhere else (e.g. `src/shared/utils`)?